utils/check_copies.py

```diff
@@ -31,6 +31,26 @@
 FULL_COPIES = {"examples/tensorflow/question-answering/utils_qa.py": "examples/pytorch/question-answering/utils_qa.py"}
 
 
+LOCALIZED_READMES = {
+    # If the introduction or the conclusion of the list change, the prompts may need to be updated.
+    # "README.md" : {
+    #    "start_prompt" : "🤗 Transformers currently provides the following architectures",
+    #    "end_prompt" : "1. Want to contribute a new model?",
+    #    "format_model_list" : "**[{title}]({model_link})** (from {paper_institutions}) released with the paper {paper_title_link} by {paper_authors}.{supplements}",
+    # },
```

Comment:
Maybe uncomment this and use it later on?

Reply:
Sure, we can uncomment this to improve consistency so that prompts of every README can be retrieved from this dict. What we can expect is that the `README.md` will always keep unchanged (no resorting) as the number of models before conversion and after conversion remains the same.